releasenotes/notes/botocore-sqs-tuples-param-cc4705bcb0647ae7.yaml

```diff
@@ -0,0 +1,5 @@
+---
+features:
+  - |
+    botocore: Enables Data Streams Monitoring for SQS when `MessageAttributes` are passed in
```

Comment:
Does this enable dsm or does it fix a specific type of error?

Reply:
oh my bad it's a fix